src/EnergyPlus/SizingManager.cc

```diff
@@ -1602,6 +1621,21 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
                                                  FinalSysSizing(AirLoopNum).AirPriLoopName,
                                                  state.dataSize->EvzMinBySysCool(AirLoopNum),
                                                  4); // Ev
+        // Ev Calculation Methodology
+        if (state.dataSize->SysSizInput(SysSizNum).SystemOAMethod == SOAM_VRP) {
+            OutputReportPredefined::PreDefTableEntry(state,
+                                                     state.dataOutRptPredefined->pdchS62svrClEvMthd,
+                                                     FinalSysSizing(AirLoopNum).AirPriLoopName,
+                                                     "Standard 62.1 Ventilation Rate Procedure");
+        } else if (state.dataSize->SysSizInput(SysSizNum).SystemOAMethod == SOAM_SP) {
+            OutputReportPredefined::PreDefTableEntry(state,
+                                                     state.dataOutRptPredefined->pdchS62svrClEvMthd,
+                                                     FinalSysSizing(AirLoopNum).AirPriLoopName,
+                                                     "Standard 62.1 Simplified Procedure");
+        } else {
+            OutputReportPredefined::PreDefTableEntry(
+                state, state.dataOutRptPredefined->pdchS62svrClEvMthd, FinalSysSizing(AirLoopNum).AirPriLoopName, "Not calculated");
+        }
```

Comment:
New output report system column that reports which methodology is used to calculate ventilation efficiency (Ev).